src/shapes/object.class.js

```diff
@@ -596,6 +596,14 @@
       ' strokeLineCap strokeDashOffset strokeLineJoin strokeMiterLimit backgroundColor clipPath'
     ).split(' '),
 
+    /**
+     * List of properties to consider for animating colors.
+     * @type Array
+     */
+    colorProperties: (
+      'fill stroke backgroundColor borderColor'
+    ).split(' '),
```

Comment:
is `borderColor` something we should consider animatable? if yes, why not also cornerColor, cornerStrokeColor and all the rest?
I would leave borderColor out for now. But of course if you think is really important, argument why.

Reply:
i personally wouldn't use either, it was just for testing so I think we can leave it out for now